source/index.txt

```diff
@@ -68,6 +68,17 @@ binaries:
 Versioning
 ----------
 
+.. important::
+
+   It is recommended you always use the latest release version of the 
```

Comment:
Can we also tell users where to find what the latest version is? 

We could link to the download center:
https://www.mongodb.com/try/download/database-tools

Reply:
Definitely, its been added!